test/uk/gov/hmrc/tai/viewModels/IncomeSourceViewModelSpec.scala

```diff
@@ -0,0 +1,280 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.tai.viewModels
+
+import controllers.FakeTaiPlayApplication
+import org.joda.time.LocalDate
+import org.scalatestplus.play.PlaySpec
+import play.api.i18n.{I18nSupport, Messages, MessagesApi}
+import uk.gov.hmrc.tai.model.domain.income.{Ceased, PotentiallyCeased}
+import uk.gov.hmrc.tai.model.domain.{JobSeekerAllowanceIncome, OtherIncome, PensionIncome, TaxedIncome}
+import utils.TaxAccountSummaryTestData
+import uk.gov.hmrc.tai.util.constants.TaiConstants.encodedMinusSign
+
+class IncomeSourceViewModelSpec
+    extends PlaySpec with FakeTaiPlayApplication with I18nSupport with TaxAccountSummaryTestData {
+
+  implicit val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
+
+  "IncomeSourceViewModel apply method (two parameter version)" must {
```

Comment:
Remove two parameter version as it is no longer relevant 
```suggestion
  "IncomeSourceViewModel apply method" must {
```